packages/testing/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@clerk/testing",
+  "version": "0.1.0",
+  "description": "Testing utils to support E2E testing with Clerk",
```

```suggestion
  "description": "Utilities to help you create E2E test suites for apps using Clerk",
```

Nit: Rewording it a bit